discordia/containers/Embed.lua

```diff
@@ -0,0 +1,35 @@
+local Snowflake = require('./Snowflake')
+
+local format = string.format
+
+local Embed, property, method = class('Embed', Snowflake)
+Embed.__description = "Represents Discord Embed."
+
+function Embed:__init(data, parent)
+	Snowflake.__init(self, data, parent)
+	self._enabled = data.enabled
+	if not self._enabled then return end 
+	self._channel_id = data.channel_id
+end
+
+function Embed:__tostring()
```

Comment:
`__tostring` methods should not return an empty string. See examples in other locations.